packages/core/src/Modules/Cashier/Components/Form/max-turnover-form.jsx

```diff
@@ -0,0 +1,123 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Formik, Form, Field } from 'formik';
+import { hasCorrectDecimalPlaces, getDecimalPlaces, routes } from '@deriv/shared';
+import { Button, Input } from '@deriv/components';
+import { Localize, localize } from '@deriv/translations';
+import { BinaryLink } from 'App/Components/Routes';
+import { connect } from 'Stores/connect';
+import { WS } from 'Services';
+
+const MaxTurnoverForm = ({ onMount, setErrorConfig, currency }) => {
+    const initial_values = {
+        max_30day_turnover: '',
+    };
+
+    const validateFields = values => {
```

Comment:
can you add a todo comment to refactor this to use the same validation and field from self exclusion?